src/components/Tooltip/Tooltip.tsx

```diff
@@ -321,6 +321,7 @@ export const Tooltip: FC<TooltipProps> = React.memo(
         top: Math.floor(y) ?? '',
         left: Math.floor(x) ?? '',
         width: width ?? '',
+        maxWidth: width ?? '',
```

Comment:
can this be decoupled from width? would gladly take a smaller tooltip when it can shrink to content, but sometimes need it to grow larger. i suspect the maxWidth value would be more useful than an explicit width

Reply:
Sure.